modules/tests/test/testconfigs/copy-template-config.go

```diff
@@ -32,8 +32,6 @@ type CopyTemplateTestConfig struct {
 }
 
 func (c *CopyTemplateTestConfig) Init(options *testoptions.TestOptions) {
-	c.TaskData.ResultTemplateName = NewTemplateName
-
 	if c.TaskData.SourceTemplateName == "" && !c.TaskData.DoNotSetSourceName {
 		c.TaskData.SourceTemplateName = testtemplate.CirrosTemplateName
```

Comment:
this can be just set from the c.TaskData.Template if we have it no? And we can also pass the desired name through SourceTemplateName/TargetTemplateName in each test case . And there would not be a need for DoNotSetSourceName / DoNotSetTargetName.

Let me know if this makes sense?